test/module/irohad/torii/torii_service_query_test.cpp

```diff
@@ -0,0 +1,142 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <gmock/gmock.h>
+
+#include "backend/protobuf/query_responses/proto_block_query_response.hpp"
+#include "backend/protobuf/query_responses/proto_query_response.hpp"
+#include "builders/default_builders.hpp"
+#include "builders/protobuf/queries.hpp"
+#include "main/server_runner.hpp"
+#include "module/irohad/torii/torii_mocks.hpp"
+#include "module/shared_model/builders/protobuf/test_query_builder.hpp"
+#include "module/shared_model/builders/protobuf/test_query_response_builder.hpp"
+#include "torii/query_client.hpp"
+#include "torii/query_service.hpp"
+#include "utils/query_error_response_visitor.hpp"
```

Comment:
`QueryErrorResponseChecker` is not used in this file, please verify that no unused includes are present.